### DISCUSSION TITLE
What exactly does inactive state of an alias mean?

### DISCUSSION THREAD

**Discussion by limonte** [anonaddy/anonaddy#393] [Q&A] [answered]
[This page](https://anonaddy.com/help/deactivating-email-aliases/) describes **how** to deactivate an alias but it doesn't answer **what** inactive alias is.

Is it so that all emails coming to inactive aliases simply won't be redirected anywhere and in fact sent to the void?

**Comment by willbrowningme** [comment, marked as answer]
The [help](https://anonaddy.com/help/) page answers this:

> **Deactivated Alias** - When an alias is deactivated, any messages sent to it will be silently discarded. The sender will not be notified of the unsuccessful delivery.
